src/main/java/com/hermesworld/ais/galapagos/topics/MessagesPerDay.java

```diff
@@ -1,6 +1,16 @@
 package com.hermesworld.ais.galapagos.topics;
 
+import lombok.Getter;
+
+@Getter
 public enum MessagesPerDay {
 
-    LOW, NORMAL, LARGE, VERY_LARGE;
+    FEW(1000), NORMAL(100000), MANY(1000000), VERY_MANY(1500000);
```

Comment:
Looks more like upper boundary. Lower boundary for fewest should be 0 (0-1000 messages).